docs/best-practices/continuous-integration.md

```diff
@@ -125,11 +125,12 @@ Save and run. The result should look like this:
 
 ![/img/best-practices/JenkinsIntegration.png](/img/best-practices/JenkinsIntegration.png)
 
-# Circle Integration
+# CircleCI Integration
```

Comment:
This will break anchor links, which is fine, but we should update all the links to this page to link to the new one 👍

Reply:
@KrauseFx Found just one reference to it here: https://github.com/fastlane/fastlane/blob/2480fdf44f4193fb5e5a044a3f22f4bed5bba918/fastlane/docs/Circle.md. Any other references I should fix after this is merged?